src/main.js

```diff
@@ -111,34 +144,28 @@ function migrateMessageFuncInternal(
 			migrateMessageFn( code, msg );
 		}
 
-		// Since patches can be disabled & enabled dynamically, we
-		// need to decide which implementation to run on each invocation.
-		finalFunc = jQuery.migrateIsPatchEnabled( code ) ?
-			newFunc :
-
-			// The function may not have existed originally so we need a fallback.
-			( origFunc || jQuery.noop );
```

Comment:
I'm not sure why I added this `noop` fallback originally, but if the patch is disabled, the function should not be available if it wasn't there originally. I added tests for that (the `push` & `isArray` ones).